Wikimate.php

```diff
@@ -1,41 +1,107 @@
 <?php
-/// =============================================================================
-/// Wikimate is a wrapper for the MediaWiki API that aims to be very easy to use.
-///
-/// @version    0.13.0
-/// @copyright  SPDX-License-Identifier: MIT
-/// =============================================================================
+/**
+ * =============================================================================
+ * Wikimate is a wrapper for the MediaWiki API that aims to be very easy to use.
+ *
+ * @package    Wikimate
+ * @version    0.13.0
+ * @copyright  SPDX-License-Identifier: MIT
+ * =============================================================================
+ */
 
 /**
  * Provides an interface over wiki API objects such as pages and files.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
  * @since   December 2010
  */
 class Wikimate
 {
 	/**
-	 * @var  string  The current version number (conforms to http://semver.org/).
+	 * The current version number (conforms to Semantic Versioning).
```

Comment:
None of the other descriptions (in this PR) use punctuation, so it seems like this shouldn't either.

Reply:
Dot dropped.